.azure-pipelines-templates/test.yml

```
@@ -5,22 +5,22 @@ parameters:
   suffix: ""
 
 steps:
-  - script: |
-      set -ex
-      ASAN_SYMBOLIZERS=(/usr/bin/llvm-symbolizer-*)
-      export ASAN_SYMBOLIZER_PATH=$(realpath ${ASAN_SYMBOLIZERS[0]})
-      ./tests.sh -VV --timeout ${{ parameters.ctest_timeout }} --no-compress-output -T Test ${{ parameters.ctest_filter }}
-    env:
-      METRICS_MONGO_CONNECTION: $(METRICS_MONGO_CONNECTION)
-      CTEST_TIMEOUT: "${{ parameters.ctest_timeout }}"
-    displayName: CTest
-    workingDirectory: build
+  # - script: |
+  #     set -ex
+  #     ASAN_SYMBOLIZERS=(/usr/bin/llvm-symbolizer-*)
+  #     export ASAN_SYMBOLIZER_PATH=$(realpath ${ASAN_SYMBOLIZERS[0]})
+  #     ./tests.sh -VV --timeout ${{ parameters.ctest_timeout }} --no-compress-output -T Test ${{ parameters.ctest_filter }}
+  #   env:
+  #     METRICS_MONGO_CONNECTION: $(METRICS_MONGO_CONNECTION)
+  #     CTEST_TIMEOUT: "${{ parameters.ctest_timeout }}"
+  #   displayName: CTest
+  #   workingDirectory: build
```

```suggestion
  - script: |
      set -ex
      ASAN_SYMBOLIZERS=(/usr/bin/llvm-symbolizer-*)
      export ASAN_SYMBOLIZER_PATH=$(realpath ${ASAN_SYMBOLIZERS[0]})
      ./tests.sh -VV --timeout ${{ parameters.ctest_timeout }} --no-compress-output -T Test ${{ parameters.ctest_filter }} -N
    env:
      METRICS_MONGO_CONNECTION: $(METRICS_MONGO_CONNECTION)
      CTEST_TIMEOUT: "${{ parameters.ctest_timeout }}"
    displayName: CTest
    workingDirectory: build
```

Looks like the partitions test relies on `./tests.sh` already running to create the venv.

---

I've done slightly differently/worse, but yes.